azext_iot/product/test/command_map.py

```diff
@@ -26,15 +26,15 @@ def load_product_test_commands(self, _):
     with self.command_group("iot product test", command_type=tests_ops) as g:
         g.command("create", "create")
         g.command("update", "update")
-        g.command("show", "show")
+        g.show_command("show", "show")
         g.command("search", "search")
     with self.command_group("iot product test case", command_type=test_cases_ops) as g:
         g.command("list", "list")
         g.command("update", "update")
     with self.command_group("iot product test task", command_type=test_tasks_ops) as g:
         g.command("create", "create")
         g.command("delete", "delete")
-        g.command("show", "show")
+        g.show_command("show", "show")
```

Comment:
whats the purpose of this show command btw

Reply:
There was a new command table custom linter rule added as of a couple days ago. It's asking for `show` commands to use `show_command` to ensure consistent behavior when encountering a missing Azure resource (like throwing an exit code of 3 if resource not found).